src/t_hash.c

```diff
@@ -2128,7 +2209,7 @@ void hincrbyfloatCommand(client *c) {
         return;
     }
     if ((o = hashTypeLookupWriteOrCreate(c,c->argv[1])) == NULL) return;
-    if (hashTypeGetValue(o,c->argv[2]->ptr,&vstr,&vlen,&ll) == C_OK) {
+    if (hashTypeGetValue(c->db, o,c->argv[2]->ptr,&vstr,&vlen,&ll) == C_OK) {
```

Comment:
if `o` was deleted from db in `hashTypeGetValue`, we will continue to use o after here.

Reply:
Nice catch. Only now i reach to make another iteration to refine the logic around it. i will fix it. thanks.